mitosheet/src/mito/components/taskpanes/Graph/graphUtils.tsx

```diff
@@ -571,16 +571,18 @@ export const getGraphElementInfoFromHTMLElement = (graphElement: Element, elemen
         const titleDivTop = mitoDivBottom - clientRect.top + 10;
         const graphDivTop = mitoDivBottom - graphOutputTop;
         let left = clientRect.left - mitoDivLeft;
+        let bottom = Math.min(titleDivTop, graphDivTop);
         // Special case for ytitle, we don't want the context menu to be on top of the title,
         // so we move it to the right a bit
         if (display === 'context-menu') {
             left += 25;
+            bottom -= 25;
```

Comment:
Would an even simpler (and probably better solution for the user) just be to set the top of this yTitle context menu in the vertical middle of the MitoContainer? It should be really easy to set that, right? 

Then it's always in a place that is reasonable for the user to look.